### ISSUE TITLE
Node-module "request" generic version

### ISSUE THREAD

**Issue by lukasT** [leukhin/co-request#15]
We had some problems when doing requests to an API. In the end we found out that it could be related to a new version of npm request (https://www.npmjs.com/package/request).

In your package.json you specify this: 
 "dependencies": {
    "request": "*"
  },

Maybe it would be more reliable if you set a static version instead of "*". 

Regards

**Comment by leukhin** [comment]
@lukasT Thats right, I think we will update our code to latest sometimes this month.

**Comment by pkoretic** [comment]
@denys any news on this one?

**Comment by idx3d** [comment]
@pkoretic hello,

Actually I don't know what version was used on the time when this lib was implemented. To make sure what version should be instead of "*" we need to test it somehow. Any ideas on that?

I understand thats sounds dumb, wasn't know that it may lead to such issue on the time.

**Comment by idx3d** [comment]
@pkoretic I believe "request": "2.69.x" should work, but cannot confirm for sure.
